x-pack/plugins/observability_shared/common/utils/get_inspect_response.ts

```diff
@@ -157,12 +158,13 @@ export function getInspectResponse({
   operationName: string;
   startTime: number;
 }): InspectResponse[0] {
-  const id = `${operationName} (${kibanaRequest.route.path})`;
+  const name = `${operationName} (${kibanaRequest.route.path})`;
+  const id = `${name} #${uuidv4().slice(0, 5)}`;
```

Comment:
perhaps use date time iso string instead of uuid to make it better id in combo box